src/mpid/ch3/include/mpid_rma_issue.h

```diff
@@ -72,68 +72,54 @@ static int init_accum_ext_pkt(MPIDI_CH3_Pkt_flags_t flags,
                               MPIR_Datatype* target_dtp, intptr_t stream_offset,
                               void **ext_hdr_ptr, MPI_Aint * ext_hdr_sz, int *flattened_type_size)
 {
-    MPI_Aint _total_sz = 0;
-    void *flattened_type;
+    MPI_Aint _total_sz = 0, stream_hdr_sz = 0;
```

Comment:
I feel it is better to rename the function to `init_stream_dtype_ext_pkt`, as it is used in both acc and gacc packets.

Reply:
It's still only for accumulate-style functions.  PUT/GET don't need it.

Reply:
`init_stream_dtype_ext_pkt` looks more straightforward to me. But you can keep it if you prefer.

Reply:
I don't really care enough to argue against it.  I'll change it.